views/partials/beerpartial.hbs

```diff
@@ -0,0 +1,20 @@
+    <div>
+      <img src='{{image_url}}' alt='{{name}}' class='beer-img' />
+    </div>
+
+    <div>
+      <p>{{name}}</p>
```

Comment:
You're missing the `<a>` tag wrapping to name to make it clickable and send you to the beer details page